### ISSUE TITLE
Intent components

### ISSUE THREAD

**Issue by ptbrowne** [cozy/cozy-ui#235]
Components that can open intents with default styling would be useful.

* `IntentButton`

Already been implemented in Bank and Drive

```bash
$ cozy   rg 'IntentButton'
bank/src/components/IntentButton/index.js
1:import IntentButton from './IntentButton'
4:export { IntentButton }
6:export default IntentButton

bank/src/components/IntentButton/IntentButton.jsx
10:class IntentButton extends React.Component {
50:export default IntentButton

drive/src/drive/components/Intent.jsx
9:class IntentButton extends React.Component {
74:export { IntentButton }
```

* `IntentModal`

Should take the 80% of the screen in desktop and 100% on mobile

See https://gitlab.cozycloud.cc/labs/cozy-bank/blob/master/src/components/IntentButton/Intent.styl

**Comment by ptbrowne** [comment]
Will be closed by https://github.com/cozy/cozy-ui/pull/243

**Comment by ptbrowne** [comment]
Closed by #243
